src/democracyworks/smartystreets/api/us_zipcode.clj

```diff
@@ -0,0 +1,47 @@
+(ns democracyworks.smartystreets.api.us-zipcode
+  "Client for the SmartyStreets US Zip Code API
+
+  https://smartystreets.com/docs/cloud/us-zipcode-api"
+  (:require
+   [democracyworks.smartystreets.core :as smartystreets]))
+
+(def base-url
+  "https://us-zipcode.api.smartystreets.com/lookup")
+
+(defn parse-response
```

Comment:
For [more about the response we're parsing](https://smartystreets.com/docs/cloud/us-zipcode-api#http-response-output).